lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/runner/Runner.scala

```diff
@@ -322,8 +328,3 @@ class Runner(
     }
   }
 }
-
-object Runner {
-
-  private val LAUNCHER_ENV_NAME = "ENSO_LAUNCHER"
```

Comment:
I don't think there is any need to support `ENSO_LAUNCHER=shell` when there is `--jvm` option in `project-manager`. Removing the [ENSO_LAUNCHER check](https://github.com/enso-org/enso/pull/12515/files#r2128189509).